src/main/java/com/microtripit/mandrillapp/lutung/MandrillApi.java

```diff
@@ -22,6 +22,8 @@
  * @since Mar 17, 2013
  */
 public class MandrillApi {
+	public static final String rootUrl = "https://mandrillapp.com/api/1.0/";
```

Comment:
This is public because we need to default it here as well as in the other classes for backwards compatibility reasons. See below comment.

Reply:
ok